### DISCUSSION TITLE
Commit to master: how to tell whether changes are (still) in master

### DISCUSSION THREAD

**Discussion by grahamperrin** [community/community#22620] [New to GitHub]
<p>The screenshot below, for example. </p>
<p>To the left:</p>
<ul>
<li><a href="https://github.com/MrAlex94/Waterfox/commit/b1231ac8b9fe6a2aa639142966a1e1ac66408e6b" rel="nofollow noopener">b1231ac</a></li>
<li>a commit to master, apparently.</li>
</ul>
<p>To the right:</p>
<ul>
<li><a href="https://github.com/MrAlex94/Waterfox/blob/master/dom/file/nsHostObjectProtocolHandler.cpp" rel="nofollow noopener">nsHostObjectProtocolHandler.cpp</a></li>
<li>the file <em>at master</em> <strong>without</strong> the changes.</li>
</ul>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/bbb312315171ab833755891bf0bea93ab67fe03d.png" data-download-href="/uploads/short-url/qMsWAqL8yboewMaDBLUjpIcV5Vz.png?dl=1" title="2018-10-06 13_31_08.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/192271/181072194-5e10580d-df50-429a-84f1-d78457599a55.png" alt="2018-10-06 13_31_08.png" data-base62-sha1="qMsWAqL8yboewMaDBLUjpIcV5Vz" width="690" height="236" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/bbb312315171ab833755891bf0bea93ab67fe03d_2_690x236.png, https://aws1.discourse-cdn.com/github/optimized/1X/bbb312315171ab833755891bf0bea93ab67fe03d_2_1035x354.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/bbb312315171ab833755891bf0bea93ab67fe03d_2_1380x472.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/bbb312315171ab833755891bf0bea93ab67fe03d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2018-10-06 13_31_08.png</span><span class="informations">2624×900 396 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>I can visually compare things, but it’s tedious.
<p>Is there an easier way to tell whether changes from a commit to master are still in master?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/grahamperrin">@grahamperrin</a>,</p>
<p>Thank you for being here! You can compare the state of your repository across branches, tags, commits, and time periods. Check out how at <a href="https://git.io/fx83S" rel="nofollow noopener">https://git.io/fx83S</a>. I hope this helps!</p>
<p>Best,</p>
<p>Andrea</p>

**Comment by grahamperrin** [comment]
<p>Thanks, but that help page confuses me; marking this topic as <em>Solved</em> was premature.</p>
<p>Please, can you apply the relevant part of that page to the example in my opening post?</p>

**Comment by grahamperrin** [comment]
<p>For the example in the opening post, should <a href="https://github.com/MrAlex94/Waterfox/compare/b1231ac...9c2c0e0" rel="nofollow noopener">https://github.com/MrAlex94/Waterfox/compare/b1231ac…9c2c0e0</a> (<a href="https://github.com/MrAlex94/Waterfox/commit/9c2c0e0c5593abb40b6093d9b2a28b294b2b0ce9" rel="nofollow noopener">9c2c0e0</a> committed five days ago) a starting point of some sort?</p>
<p>For <a href="https://github.com/MrAlex94/Waterfox/compare/b1231ac...9c2c0e0#files_bucket" rel="nofollow noopener">https://github.com/MrAlex94/Waterfox/compare/b1231ac…9c2c0e0#files_bucket</a> I get: </p>
<p>&gt; … too large to display on GitHub.</p>
